vtex-apps/search-result
# fix: slugified not necessary since we already have it from facets API

#### What problem is this solving?

CSS handles were not working when using diacrisis, since we already have the slugified value in facets API I just changed following the same as here:

https://github.com/vtex-apps/search-result/blob/8f0a65ca0c230dc1061d18ef282f8ae8cb5ffaf6/react/components/FacetItem.js#L47


#### How to test it?

Go to mobile version and check the facets with diacrisis, you will see that facets with diacrisis are without the handle filterItem with the expected slugified value

In the Example URL's: Mobile version => Filters => Categorie, inspect

[After](https://iespinoza--toffro.myvtex.com/barbati/incaltaminte)

![image](https://user-images.githubusercontent.com/13649073/224813330-181fd46b-a4ba-4f24-8875-9a684137902f.png)


[Before](https://master--toffro.myvtex.com/barbati/incaltaminte)

![image](https://user-images.githubusercontent.com/13649073/224813749-0c40e451-c257-4f46-ad01-53f1d360f9f6.png)

#### Related to / Depends on

Zendesk: 756986

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/yTSPaYi0GEJvR2DKLd/giphy.gif)
